ci_framework/roles/hive/molecule/default/converge.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  vars:
+    cifmw_hive_kubeconfig: test_path
+    cifmw_hive_platform: openstack
+    cifmw_hive_action: claim
+    cifmw_hive_openstack_pool_name: test-pool
+    cifmw_hive_namespace: test-namespace
+    claim_name: test-claim-by-molecule
+  roles:
```

Comment:
Since this is not going to work for now, I think that you can do:
- on all ansible.builtin.shell and ansible.builtin.command calls, you can a check_mode, so we can skip this commands in molecule:
```
      ansible.builtin.command:
        cmd: my_awesome_command
      check_mode: "{{ hive_dryrun | default(omit) }}"
```